traffic_portal/app/src/common/modules/form/deliveryService/edit/FormEditDeliveryServiceController.js

```diff
@@ -161,6 +166,30 @@ var FormEditDeliveryServiceController = function(deliveryService, origin, topolo
 	};
 
 	$scope.save = function(deliveryService) {
+		if (deliveryService.sslKeyVersion !== null && deliveryService.sslKeyVersion !== 0 &&
```

Comment:
I dont think this page will ever have deliveryService.sslKeyVersion !== null. I think that /deliveryservices endpoint doesnt return the ssl key version.  I dont think we know that until we are onto the ssl keys page itself.  TP still fails to updated the DS but just returns a Forbidden message

Reply:
From my testing, the ssl_key_version on the deliveryservice table is always updated when the ssl keys are changed/created. Also the /deliveryserivces endpoint does return that field as `sslKeyVersion`. Inspecting the network call in TP when loading the DS table, each DS has it's `sslKeyVersion` returned.

Reply:
interesting. maybe cause i was just running TO and TP locally. ill try again in a better environment

Reply:
my bad. definitely just my setup. looks good in ciab.